test/utils/template.test.ts

```diff
@@ -0,0 +1,7 @@
+import { toSubmoduleFiles } from "./template";
+
+describe("Templating test utility tests", () => {
+    test("To submodule outputs correct content", () => {
+        expect(toSubmoduleFiles(["test.md"])).toContain("third_party/submodule/test.md");
+    });
+});
```

Comment:
Codacy found an issue: [file should end with a newline](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6092258)